### DISCUSSION TITLE
Procedure for re-claiming a node on cloud

### DISCUSSION THREAD

**Discussion by lerilar** [netdata/netdata#15181] [Q&A]
I am running TrueNAS Scale and had initially the TrueNAS Scale app (container) running, but decided to delete that and run the kickstart on debian instead. 

It worked great and I claimed the node on Cloud, but when the server restarted I had some permission issues and had to reinstall. Since that I have had no luck in getting the server to appear on cloud. I tried to reclaim and had at one point three inactive nodes showing - but no data.

What is the bullet-proof method of getting the server bac into cloud (the local 19999 is working perfectly).

**Comment by andrewm4894** [comment]
If you add `--claim-id ${uuidgen}` to the netdata cloud install/claim command it will try to do a fresh claim using a new claim id generated by uuidgen

**Comment by lerilar** [comment]
--claim-id "$(uuidgen) worked perfectly.  Thank you. I probably would have tried that sooner if I didn't have such a hard time figuring out how to remove the nodes from the cloud. All good now. The error text returned from the unsuccessful claim was spot on!
